app.js

```diff
@@ -76,7 +76,7 @@ app.get("/", (req, res) => {
 });
 
 app.use("/api/v1/users", userRouter); //users route
-app.use("/api/v1/universities", universityRouter); //univesity route
+app.use("/api/v1/universities", authenticate, universityRouter); //univesity route
```

Comment:
Move the `authenticate` out of here because not all university routes will be needing it, add it to the specific routes that need it. For instance, the get route will not be needing it. I a situation you need all route to a particular resource to have it , you can as well add it to the route file and not the entry app.js